leverage/modules/auth.py

```
@@ -2,13 +2,13 @@
 from pathlib import Path
 from configparser import NoSectionError, NoOptionError
 
-import hcl2
 import boto3
-from configupdater import ConfigUpdater
+import hcl2
```

Ruff (0.8.2)

`hcl2` imported but unused

Remove unused import: `hcl2`

(F401)